portal/app/[locale]/_components/navbar/_components/tvl/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { useTvl } from 'hooks/useTvl'
+import { useTranslations } from 'next-intl'
+import React from 'react'
+import Skeleton from 'react-loading-skeleton'
+
+import { Amount } from './amount'
+import { DollarSign } from './dollarSign'
+import { HemiLogo } from './hemiLogo'
+
+export const Tvl = function () {
+  const { data, isError, isPending } = useTvl()
+  const t = useTranslations('navbar')
+
+  const getAmount = function () {
+    if (isError) {
+      return '-'
+    }
+
+    return new Intl.NumberFormat('en', {
+      compactDisplay: 'short',
+      maximumFractionDigits: 2,
+      notation: 'compact',
+    }).format(data)
+  }
+
+  return (
+    <section
+      className="shadow-soft h-22 relative mb-4 mt-4 w-full overflow-hidden rounded-lg
+      md:mb-0"
+      style={{
+        background:
+          'linear-gradient(0deg, #262626, #262626),linear-gradient(0deg, #FFFFFF, #FFFFFF),linear-gradient(180deg, rgba(0, 0, 0, 0) 10.65%, rgba(0, 0, 0, 0.6) 76.58%)',
+      }}
+    >
+      <HemiLogo className="absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2 transform" />
+      <div className="absolute left-0 top-0 w-full">
+        <div className="flex flex-col gap-y-2 p-4">
+          <div className="flex w-full items-center justify-between">
+            <span className="font-inter-variable text-sm font-medium text-white">
+              {t('tvl')}:
+            </span>
+            <DollarSign />
+          </div>
+          <div className="-translate-x-1">
+            {isPending ? (
+              <Skeleton baseColor="#262626" highlightColor="#303030" />
+            ) : (
+              <Amount value={`$${getAmount()}`} />
```

Comment:
_I think_ that if there's an error, the `$` should not be shown - just the `-`. See example on Stake

<img width="1610" alt="image" src="https://github.com/user-attachments/assets/c5703017-df9e-459a-8b8f-a285114fa1c5" />